scripts/realms/reformatRealms.js

```diff
@@ -21,13 +20,31 @@ const output = {
   TW: reformatFile('./data/TW.json'),
   US: reformatFile('./data/US.json'),
   CN: reformatFile('./data/CN.json'),
+  classicEU: reformatFile('./data/classicEU.json'),
+  classicKR: reformatFile('./data/classicKR.json'),
+  classicTW: reformatFile('./data/classicTW.json'),
+  classicUS: reformatFile('./data/classicUS.json'),
 };
 
 fs.writeFileSync(
   './output.js',
   `// Generated file, changes will be overwritten!
 // eslint-disable
 
-export default ${JSON.stringify(output)};
+export default {
+${Object.entries(output)
+  .map(
+    ([key, value]) =>
+      `  ${key}: [\n${value
+        .map(
+          (realm) =>
+            `    { name: ${
+              realm.name.includes("'") ? '"' + realm.name + '"' : "'" + realm.name + "'"
```

Comment:
could this be simplified by mapping the names when generating `output` and then using stringify?

Reply:
I'm not sure how. I tried all sorts of combinations with JSON stringify but I kept getting the keys output as strings.

This can be less complex if we are willing to change the format from:
`{ name: 'Aegwynn', slug: 'aegwynn' },`
to:
```
{
 name: 'Aegwynn',
 slug: 'aegwynn' 
},
```

Reply:
i'm fine with reformatting the output. can always run prettier on it after

Reply:
ok, I guess I was overthinking it. I didn't realize the commit hook would fix the formatting.
I changed this back and updated the README to indicate this.